docker/README.md

```diff
@@ -0,0 +1 @@
+# Matee CI for Android
```

Comment:
Pls add more info - imagine our CI machine goes up in smoke, and we need to configure everything again on a new machine